types/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@sap-devx/bas-platform-types",
   "publisher": "SAPOSS",
-  "version": "0.0.33",
+  "version": "0.0.13",
```

Comment:
Is it by matter?

Reply:
Yes. Yes. Later we will decide both when to bump it to 1.0.0